libraries/Microsoft.Bot.Builder.LUIS/RecognizerResultExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Microsoft.Bot.Builder.LUIS
+{
+    public static class RecognizerResultExtensions
+    {
+        public static Tuple<string, double> GetTopScoringIntent(this RecognizerResult result)
```

Comment:
Consider using the more modern Tuple syntax, to give better compiler hints about things. 
https://blogs.msdn.microsoft.com/seteplia/2017/11/01/dissecting-the-tuples-in-c-7/

public static (string key, double score) GetTopScoringIntent(this RecognizerResult result)

Reply:
Done, wasn't sure they were supported on the build machine